### DISCUSSION TITLE
Suggestion: "For You" based on contributions.

### DISCUSSION THREAD

**Discussion by Bigjango13** [community/community#14490] [Other Feature Feedback, Questions, & Ideas]
In my understanding "For You" recommends repos and people based on what I have stared, the people I am following and what the people I am following have stared.
This is good for finding repos that I would like to use, but not very good at finding repos that I would like to contribute to.
I think it would be much better if "For You" recommended repos based on contribution history.

**Comment by eggplantlwj** [comment]
Agree！
